README.md

```diff
@@ -52,6 +52,11 @@ Then install what you just cloned.
 
 `pip install -e <path-to-platoon-folder>`
 
+### Attention
+Install ordering for dependencies:
+
+1. [NCCL](https://github.com/NVIDIA/nccl) should be installed first.
+2. [libgpuarray/ pygpu](https://github.com/Theano/libgpuarray) should be installed after the installation of NCCL.
```

Comment:
I got this idea from [this issue](https://github.com/mila-udem/platoon/issues/72). I followed the instruction from nouiz , and it worked. I'm not sure why, but @nouiz might know it.